### DISCUSSION TITLE
Feature requests

### DISCUSSION THREAD

**Discussion by InsertDisc** [InsertDisc/pattrmm#3] [Ideas]
Any feature requests or ideas for improvements? Post them here.

**Comment by alejakun** [comment]
I think the ability to distinguish between Weekly releasing shows and shows with all episodes aired simultaneously (Netflix style).

**Reply by InsertDisc** [reply]
Can you clarify what you're asking for? What do you mean distinguish between the two?

**Reply by alejakun** [reply]
I mean, to have different banners for shows that start airing several episodes at the same time (for example most Netflix shows start airing half season or the full season the same day). In contrast some shows airs a new episode every week (for example AppleTV+ or HBOmax release one episode every 1-2 weeks), the idea would be to have a "Airing" (what it's been labeled now) vs "Airing Weekly" for shows expecting episodes for the next several weeks.

**Reply by InsertDisc** [reply]
I'm not completely against the idea but I'm not quite sure how I would filter that at the moment.

**Reply by itsFelixH** [reply]
I am also interested in this ability!
There are 3 differnt release forms I can think of:

- releasing one episode after each other (usually once a week but it can differ) -> "Airing Weekly"
- releasing the whole season (old netflix style) -> probably just "Airing"
- releasing the season in parts (new netflix style) -> probably just "Airing"

I'd like to extend your idea. This distinguishment could also be added to the returning lablel. Maybe instead of the "Returning 12/22" label it can be:

- weekly release -> "New episodes 12/22"
- season release -> "New season 12/22"
- release in parts ->"New Part 12/22"

I will try to think about if and how it is possible with the current API.

**Comment by tylerreed19** [comment]
The ability to show the next airing date of an episode. Not returning date. Text would say something like "Airing 12/22" then the 23rd it'd say "Airing 12/29"

**Reply by itsFelixH** [reply]
Hey, just for clarity and understanding your request: So the show is returning 12/22. After that date, when the first episode dropped, you want the season poster to show airing 29/12, since this is the date, where the next episode drops?
So your request means if a show is returning & the season has started airing, show the next airing date in the banner instead of just showing "airing"?

**Reply by tylerreed19** [reply]
Yes essentially. Maybe a 2nd option, the first being the default, if a show is Airing it just says Airing on the Overlay. And my idea was Airing 12/28 (Whatever date) then on 12/29 I run the script again and it updates it to the next episodes air date, a week later or whatever it may be. Airing 01/04. 

It would be really useful for users like me who don't have Sonarr or something else setup to auto download. It'd be a great reminder instead of checking my trakt calendar when I remember to.

**Reply by tylerreed19** [reply]
And personally I would prefer it on the Show itself poster. The Airing 12/28, and if possible the Season Poster as well. But I don't want to make this more complicated. Maybe a choice to have it on Show or Season posters or Both. But personally I'd prefer the show poster to have the Overlay more than the Season.
